src/dotnet-run/project.json

```diff
@@ -9,14 +8,14 @@
 
         "Microsoft.Net.Compilers.netcore": "1.2.0-beta1-20160108-01",
         "System.Net.Sockets": "4.1.0-rc3-23727",
-        "System.CommandLine" : "0.1.0-d111815-3",
+        "System.CommandLine" : "0.1.0-e160119-1",
 
         "Microsoft.DotNet.ProjectModel": "1.0.0-*",
         "Microsoft.DotNet.Cli.Utils": "1.0.0-*"
     },
     "frameworks": {
         "dnxcore50": {
-            "imports": "portable-net451+win8"
+            "imports": "portable-net45+win8+wpa81+aspnetcore50"
```

Comment:
Is this required? AFAIK `portable-net45+win8` should be enough. You just have to capture a subset of the items, and you can only choose one subset right now, so `portable-net45+win8` is the broadest set :).